content/articles/transferring-domain-between-accounts.md

```diff
@@ -15,46 +15,57 @@ categories:
+</info>
 
-You can transfer a domain between DNSimple accounts any time via the DNSimple interface. Billing responsibility will be transferred to the new domain account once the move is accepted. Moving a domain to another account will also transfer all associated resources, like DNS records, SSL certificates, and email forwards.
+You can transfer a domain between DNSimple accounts any time via the DNSimple interface. Billing responsibility will be transferred to the new domain account once the move is accepted. Transferring a domain to another account will transfer all associated resources, like DNS records, SSL certificates, and email forwards.
 
 Working with a Reseller? Follow the steps outlined [here](#accepting-a-transfer) to gain access to your account.
 
 <warning>
-Once the transfer is accepted, you'll no longer be able to manage the domain.
+Once the transfer is accepted, anyone who previously had access to the domain in your account will no longer be able to manage the domain.
 
-When transferring a domain to another DNSimple account, the domain's registrant information will be updated to reflect the new account. This may result in the domain being [locked from further transfers for 60 days](/articles/icann-60-day-lock-registrant-change/).
+When transferring a domain to another DNSimple account, the domain's registrant information will be updated to reflect the new account. This may result in the domain being [locked from external transfers for 60 days](/articles/icann-60-day-lock-registrant-change/).
 </warning>
 
 ## Initiating a transfer
 
-To transfer a domain to another DNSimple account, go to the domain's management page, click on the _Settings_ tab and then follow the <label>Transfer ownership</label> link.
+1. Click on the relevant domain. 
+1. Click the **Settings** tab on the left side.
+1. In the **Share or transfer domain** card, click **Transfer**.
 
-![screenshot: Initiate domain transfer](/files/transfer-ownership.png)
+    <!--- needs screenshot -->
 
-Enter the destination email address of another DNSimple account. Make sure the email address you enter matches the correct account. Once the domain has been transferred, you won't be able to control this domain under your current account.
+1. Enter the destination email address of another DNSimple account.
+1. Click **Move domain**.
+1. A banner will appear confirming the request to push the domain. 
 
 <warning>
-Make sure to use the [account email address](/articles/changing-account-email/), rather than a [user email address](/articles/changing-user-email/). Otherwise, the push will not be successful.
+Make sure to correctly enter the account email address. Once the domain has been transferred, you won't be able to control it under your current account.
```

Comment:
I would rewrite this to make it clear that when they start the transfer and if they write the email wrong, they won't be able to cancel the transfer on their end.

> Be sure to enter the correct account email address, as you won't be able to cancel the transfer on your own if the email is incorrect. If the email is wrong and you need to cancel the transfer, you will have to contact support for assistance.

Reply:
I agree with Carla that we should add this.